engine/src/flutter/shell/platform/embedder/embedder_semantics_update.h

```diff
@@ -66,6 +66,7 @@ class EmbedderSemanticsUpdate2 {
   std::vector<FlutterSemanticsNode2> nodes_;
   std::vector<FlutterSemanticsNode2*> node_pointers_;
   std::vector<FlutterSemanticsCustomAction2> actions_;
+  std::vector<FlutterSemanticsFlags> flags_;
```

Comment:
nit: This is safe as-is, but I'd put this at the end to avoid splitting up `actions_` and `action_pointers_`.